ui-new/packages/tidb-dashboard-lib/src/apps/Overview/index.tsx

```diff
@@ -1,40 +1,22 @@
 import React, { useContext } from 'react'
-import { Col, Row } from 'antd'
 import { HashRouter as Router, Routes, Route } from 'react-router-dom'
 
 import { Root } from '@lib/components'
 import { addTranslations } from '@lib/utils/i18n'
 
-import MonitorAlert from './components/MonitorAlert'
-import Instances from './components/Instances'
-import Metrics from './components/Metrics'
-
 import translations from './translations'
 import { OverviewContext } from './context'
 import { useLocationChange } from '@lib/hooks/useLocationChange'
+import { List, Detail } from './pages'
 
 addTranslations(translations)
 
-function Overview() {
-  return (
-    <Row>
-      <Col span={18}>
-        <Metrics />
-      </Col>
-      <Col span={6}>
-        <Instances />
-        <MonitorAlert />
-      </Col>
-    </Row>
-  )
-}
-
 function AppRoutes() {
   useLocationChange()
-
   return (
     <Routes>
-      <Route path="/overview" element={<Overview />} />
+      <Route path="/overview" element={<List />} />
+      <Route path="/overview/detail" element={<Detail />} />
```

Comment:
I think these 2 pages are not `List` and `Detail` relationship, maybe `Overview` and `Detail`?